data.sql

```diff
@@ -5,3 +5,14 @@ VALUES
   (2, 'Gabumon', '2018-11-15', 2, true, 8),
   (3, 'Pikachu', '2021-01-07', 1, false, 15.04),
   (4, 'Devimon', '2017-05-12', 5, true, 11);
+
+  /* Additional insertions */
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutured, weight_kg)
+VALUES
+  (5, 'Charmander', '2020-02-08', 0, true, 11),
+  (6, 'Plantmon', '2021-11-15', 2, true, 5.7),
+  (7, 'Squirtle', '1993-04-02', 3, false, 12.13),
+  (8, 'Angemon', '2005-06-12', 1, true, 45),
+  (9, 'Boarmon', '2005-06-07', 7, true, 20.4),
+  (10, 'Blossom', '1998-10-13', 3, true, 17),
+  (11, 'Ditto', '2022-05-14', 4, true, 22);
```

Comment:
- [ ] *[Optional]* You have done a great job implementing the previous reviewer's suggestions 👏 However, in the inserted data you forgot to use the exact data given in the project requirements, 

Example: 
   - in the provided data, instead of adding `Plantmon', '2021/11/15', '2', 'true', '-5.7'` you added `(6, 'Plantmon', '2021-11-15', 2, true, 5.7)` 😨 Please try to enter the provided data as it is to get the expected result 🙏 
   
   - Also the `id` column automatically generates unique values for each row, so you don't need to explicitly insert values into the "id" column when inserting data into the table. 🙏